### PR TITLE
gotop 3.0.0 (new formula)

### PR DESCRIPTION
**Pull request by dawidd6**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

https://github.com/cjbassi/gotop

### CODE REVIEW

**Review by chenrui333 — APPROVED** [review]
👍

File: Formula/gotop.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  sha256 "d5147080bb6057f0bf0900b38438e89aa066959c845bdd4c84a9c9fe478b176f"
+
+  depends_on "go" => :build
+
+  def install
+    system "go", "build", "-mod=vendor", "-ldflags", "-s -w", "-trimpath", "-o", bin/name
+  end
+
+  test do
+    assert_equal version.to_s, shell_output("#{bin}/gotop --version").chomp
```
Comment (chenrui333) [inline Formula/gotop.rb]: might be better to use `expect` syntax to improve the test part.

Comment (dawidd6) [reply Formula/gotop.rb]: It's a TUI program, can't really test much here.

Comment (chenrui333) [reply Formula/gotop.rb]: 👍

**Pull request closed without merging**